src/Cake.Core/Scripting/Processors/UriDirectiveProcessor.cs

```diff
@@ -57,13 +60,22 @@ public sealed override bool Process(IScriptAnalyzerContext context, string line,
             return false;
         }
 
+        private bool IsUriFromLegacyPattern(string[] tokens)
```

Comment:
Can we move this method below `ParseUriFromTokens`?

Reply:
Done